cloud/scope/powervs_machine_test.go

```diff
@@ -190,6 +191,254 @@ func TestNewPowerVSMachineScope(t *testing.T) {
 	}
 }
 
+func TestGetServiceInstanceID(t *testing.T) {
```

Comment:
For some reason we have same method defined with clusterscope as well for reference https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/01e1771ca8cbf8b9969bb64b6956b3d470219313/cloud/scope/powervs_cluster.go#L384, For which I have tried adding UT with this [PR](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1927/commits/759f87606596ead820673a589e9b0b3dddee1ef4)

What I thought is since we are testing GetServiceInstanceID no need to create actual machine scope like scope := setupPowerVSMachineScope(..)

Reply:
Done